source/connect/tls-ssl-connection.txt

```diff
@@ -11,3 +11,106 @@ TLS/SSL Connection Tab
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+The :guilabel:`TLS / SSL` tab allows you to connect deployments via TLS / SSL.
+For more information on :guilabel:`TLS / SSL`, see :manual:`TLS Options 
+</reference/connection-string/#tls-options>`
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`TLS / SSL` tab.
+
+      You can either use a ``Default`` connection or to set the TLS / SSL 
+      connection ``On`` or ``Off``. 
+
+      .. list-table::
+         :header-rows: 1 
+         :widths: 50 50
+      
+         * - Option
+           - Description
+
+         * - Default
+           - The ``Default`` option does not use the ``TLS / SSL`` connection option. 
+             When using the ``Default`` option, you can specify additional certificate 
+             options. To see more on the additional options available, see 
+             :ref:`<additional-tls>`.
+      
+         * - On
+           - The ``On`` option is enabled when using a :manual:`DNS seedlist (SRV) 
+             </reference/connection-string/#std-label-connections-dns-seedlist>` in 
+             the connection string. When TLS / SSL Connection is ``On``, you can 
+             specify additional certificate options for your connection string. 
+             To see more on the additional certificate options available, see 
+             :ref:`<additional-tls>`.
+
+             .. note::
```

Comment:
Thanks for adding this note. This is helpful. I'm curious why Compass doesn't disable the option if you're using a non-DNS seedlist string? Curious to ask engineering about this.

Reply:
Agreed will follow up further on this with engineering.